spec/graphql/language/generation_spec.rb

```diff
@@ -30,7 +31,7 @@
     describe "inputs" do
       let(:query_string) {%|
         query {
-          field(int: 3, float: 4.7e-24, bool: false, string: "☀︎🏆\\n escaped \\" unicode ¶ /", enum: ENUM_NAME, array: [7, 8, 9], object: {a: [1, 2, 3], b: {c: "4"}}, unicode_bom: "\xef\xbb\xbfquery")
+          field(int: 3, float: 4.7e-24, bool: false, string: "☀︎🏆\\n escaped \\" unicode ¶ /", enum: ENUM_NAME, array: [7, 8, 9], object: { a: [1, 2, 3], b: { c: "4" } }, unicode_bom: "\xef\xbb\xbfquery")
```

Comment:
They were so that GraphQL::Language::Generation and GraphQL::Schema::Printer print value literals the same way.  That way the GraphQL::Language::Parser "parses the test schema" test can assert that no information is lost through during parsing by simple doing

```
        schema_string = GraphQL::Schema::Printer.print_schema(schema)
        document = GraphQL::Language::Parser.parse(schema_string)

        assert_equal schema_string, document.to_query_string
```